wurst/objects/abilities/beastmaster/Druid/Rejuvenation.wurst

```diff
@@ -0,0 +1,64 @@
+package Rejuvenation
+
+// Standard Library Imports:
+import AbilityObjEditing
+import ObjectIds
+import ClosureEvents
+import ClosureTimers
+
+// Local Imports:
+import LocalObjectIDs
+import ToolTipsUtils
+import HealingSystem
+
+
+let COOLDOWN = 30.
+let MANA_COST = 30
+let HEAL_AMOUNT = 150.
+let HEAL_TICKS = 3
+let DURATION = 10.
+let CAST_RANGE = 15.
+let BUTTON_POS_X = 2
+let BUTTON_POS_Y = 0
+let HOTKEY = "E"
+let NAME = "Rejuvenation"
+let TOOLTIP_BASIC = "Rejuvenation"
+let TOOLTIP_EXT   = "Heal target ally for " + HEAL_AMOUNT.toToolTipGreen() + " over " + DURATION.toToolTipOrange()
+                    + " seconds. " + makeToolTipCooldown(COOLDOWN)
+
+@compiletime function createRejuvenationAbility() returns AbilityDefinitionRejuvination
+    return new AbilityDefinitionRejuvination(ABILITY_REJUVENATION)
+        ..setName(NAME)
+        ..presetManaPointsGained(LVL -> 0)
+        // This cannot be 0 or the spell won't cast.
+        ..presetHitPointsGained(LVL -> 0.1)
+        ..presetAllowWhenFull(LVL -> 1)
+        ..presetDurationHero(LVL -> DURATION)
+        ..presetDurationNormal(LVL -> DURATION)
+        ..presetCooldown(LVL -> COOLDOWN)
+        ..presetManaCost(LVL -> MANA_COST)
+        ..presetCastRange(LVL -> CAST_RANGE)
+        ..presetButtonPosNormal(BUTTON_POS_X, BUTTON_POS_Y)
+        ..presetHotkey(HOTKEY)
+        ..presetTooltipNormal(LVL -> TOOLTIP_BASIC)
+        ..presetTooltipNormalExtended(LVL -> TOOLTIP_EXT)
+        ..presetTargetsAllowed(lvl -> commaList(
```

Comment:
No need to set up this field, the values are exactly the same except for the nonacient part, which doesn't matter as the only ancient units are bosses and they aren't on your team.